antispam_bee.php

```diff
@@ -2412,14 +2412,10 @@ public static function delete_spam_reason_by_comment( $comment_id ) {
 	 * Updates the Antispam Bee reason for manual transitions
 	 *
 	 * @since   2.9.2
-	 * @param  [int] $comment Comment Object or comment ID.
+	 * @param  [obj] $comment Comment Object.
```

Comment:
@param  ~[obj]~ $comment Comment Object.
@param  WP_Comment $comment Comment Object.

It would be good, if it would be called a WP_Comment. The brackets are not needed.